src/main/java/com/exasol/repository/GitHubGitRepository.java

```diff
@@ -0,0 +1,52 @@
+package com.exasol.repository;
+
+import java.io.IOException;
+import java.util.*;
+
+import org.kohsuke.github.GHRelease;
+import org.kohsuke.github.GHRepository;
+
+import com.exasol.github.GitHubException;
+
+/**
+ * A GitHub-based repository.
+ */
+public class GitHubGitRepository implements GitRepository {
+    private final GHRepository repository;
+    private final Map<String, GitRepositoryContent> gitRepositoryContents = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, GitRepositoryContent> gitBranchContents = new HashMap<>();
```
Naming is important here otherwise the fact that this is a map, is confusing.

Reply:
done